feed-parsers/class-feed-parser-activitypub.php

```diff
@@ -686,6 +686,21 @@ private function disable_polling( User_Feed $user_feed ) {
 		$user_feed->update_metadata( 'next-poll', gmdate( 'Y-m-d H:i:s', time() + YEAR_IN_SECONDS ) );
 	}
 
+	public function get_activitypub_actor( $user_id ) {
+		if ( null === $user_id ) {
+			$user_id = Friends::get_main_friend_user_id();
+			if ( class_exists( '\ActivityPub\Collection\Followers' ) ) {
```

Comment:
is this to check if the ActivityPub plugin is installed? If so, I would suggest using `\define( 'ACTIVITYPUB_PLUGIN_VERSION' );` instead!

Reply:
Ah, oops, I just reused some code from where I get the followers. Thanks.